root/area/edit_form.tt

```diff
@@ -5,21 +5,21 @@
 
   <div class="half-width">
     <fieldset>
-      <legend>[%- l('Area details') -%]</legend>
+      <legend>[%- 'Area details' -%]</legend>
       [%- form_row_name_with_guesscase(r) -%]
-      [%- form_row_text_long(r, 'comment', add_colon(l('Disambiguation'))) -%]
-      [%- form_row_select(r, 'type_id', l('Type:')) -%]
-      [%- form_row_text_list(r, 'iso_3166_1', add_colon(l('ISO 3166-1')), l('ISO 3166-1')) -%]
-      [%- form_row_text_list(r, 'iso_3166_2', add_colon(l('ISO 3166-2')), l('ISO 3166-2')) -%]
-      [%- form_row_text_list(r, 'iso_3166_3', add_colon(l('ISO 3166-3')), l('ISO 3166-3')) -%]
+      [%- form_row_text_long(r, 'comment', add_colon('Disambiguation')) -%]
```

Comment:
Applicable suggestion from https://github.com/metabrainz/musicbrainz-server/pull/3120/files#r1424358324

```suggestion
      [%- form_row_text_long(r, 'comment', 'Disambiguation:') -%]
```

Reply:
There's a ton of these, but if you also agree with dropping them then I will do that :)

Reply:
I don’t mind either way, this has to be converted to React at some point anyway.